docs/guides/external-node/08_pruning.md

```diff
@@ -0,0 +1,40 @@
+# Pruning
+
+It is possible to configure ZKSync Node to periodically remove all data from batches older than a configurable
```

Comment:
Nit: The correct capitalization seems to be "ZKsync" (i.e., with lowercase "s").

Reply:
Corrected